### PR TITLE
[CARBONDATA-292] add infomation for COLUMNDICT operation

### PR DESCRIPTION
**Pull request by Jay357089**
https://issues.apache.org/jira/browse/CARBONDATA-292

add info in doc, no need to test.

### CODE REVIEW

File: docs/DML-Operations-on-Carbon.md [hunk 1]
```diff
@@ -104,8 +109,10 @@ Following are the options that can be used in load data:
                                  'MULTILINE'='true', 'ESCAPECHAR'='\', 
                                  'COMPLEX_DELIMITER_LEVEL_1'='$', 
                                  'COMPLEX_DELIMITER_LEVEL_2'=':',
-                                 'ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary'
+                                 'ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary',
+                                 'COLUMNDICT'='empno:/dictFilePath/empnoDict.csv, empname:/dictFilePath/empnameDict.csv'
```
Comment (jackylk) [inline docs/DML-Operations-on-Carbon.md]: do not give this option since it can not be used together with `ALL_DICTIONARY_PATH`

Comment (Jay357089) [inline docs/DML-Operations-on-Carbon.md]: i have give Note in the below.  if it is not enough, should i delete this option or close this pr?

Comment (jackylk) [inline docs/DML-Operations-on-Carbon.md]: No, I mean just delete it from the Example section. 
And that node should be added in the option expanation section.

Comment (Jay357089) [inline docs/DML-Operations-on-Carbon.md]: done.

File: docs/DML-Operations-on-Carbon.md [hunk 2]
```diff
@@ -91,6 +91,12 @@ Following are the options that can be used in load data:
     ```ruby
     OPTIONS('ALL_DICTIONARY_PATH'='/opt/alldictionary/data.dictionary')
     ```
+- **COLUMNDICT:** dictionary file path for single column.
+
+    ```ruby
+    OPTIONS('COLUMNDICT'='column1:dictionaryFilePath1, column2:dictionaryFilePath2')
+    Note: ALL_DICTIONARY_PATH and COLUMNDICT can't be used together.
```
Comment (jackylk) [inline docs/DML-Operations-on-Carbon.md]: Move the note out of the coding section, it should be text only, it is not code

**Comment by Jay357089** [comment]
@jackylk pls check change.

**Pull request closed without merging**